projects/user-telemetry-client/src/TelemetryService.ts

```diff
@@ -43,7 +43,8 @@ export class UserTelemetryEventSender {
     private throttledSendEvents = throttle(this.sendEvents, this.throttleDelay, { trailing: true, leading: false })
 
     public addEvent(event: IUserTelemetryEvent): void {
-      this.eventBuffer.push(event);
+      const valueToNumber = ( value: number | boolean ) => + value; 
```

Comment:
Non-blocking: this feels like a private method, rather than something that's declared each time this function is invoked.

Reply:
Fair point, it was mainly there to make it readable. I've refactored, lmk what you think.

Reply:
That does make sense. This looks fine too 👍